[ML] add windowing support for text_classification (#83989)
elastic/elasticsearch: docs/reference/ml/ml-shared.asciidoc

tokens.
end::inference-config-nlp-tokenization-bert-do-lower-case[]

tag::inference-config-nlp-tokenization-bert-span[]

> When `truncate` is `none`, this value can be set to indicate if longer text sequences

Reviewer: The first sentence made me think this was a boolean. Here's a rewrite for your consideration:
```suggestion
When `truncate` is `none`, you can partition longer text sequences
```

> should be partitioned for inference. The configured number is how many tokens overlap between each

Reviewer: More minor edits:
```suggestion
for inference. The value indicates how many tokens overlap between each
```

subsequence.

> Default value is `-1`, indicating no windowing or spanning is to be done.

Reviewer: Minor edit:
```suggestion
The default value is `-1`, indicating no windowing or spanning occurs.
```

> Note: when your typical input is just slightly larger than `max_sequence_length`, it may be best to simply truncate. As

Reviewer: ```suggestion
NOTE: When your typical input is just slightly larger than `max_sequence_length`, it may be best to simply truncate;
```
